packages/core/strapi/lib/commands/migrate.js

```diff
@@ -0,0 +1,12 @@
+'use strict';
+
+const strapi = require('../index');
+
+/**
+ * `$ strapi migrate`
+ */
+module.exports = async () => {
+  await strapi().migrate();
+  console.log('Migration completed successfully');
+  process.exit(1);
```

Comment:
Maybe better to use `process.exit()` without `1`? If the operation success why do you pass a parameter to tell a terminal that the code exit with an error?

https://nodejs.org/api/process.html#processexitcode